### PR TITLE
Catch undefined manager_type

### PR DESCRIPTION
**Pull request by core23**
This could lead to an error if you use something else than orm or mongodb as an entity manager.

### CODE REVIEW

**Comment by OskarStark** [comment]
ping @rande

**Comment by core23** [comment]
@rande Should we ignore this (return) or throw an exception?

**Comment by OskarStark** [comment]
i would throw an exception

ping @rande

**Comment by OskarStark** [comment]
any news on this?

**Comment by core23** [comment]
IMO a simple return should do it. Today there is no possibility to run this line: https://github.com/sonata-project/SonataUserBundle/blob/master/DependencyInjection/Configuration.php#L63

**Comment by OskarStark** [comment]
ok :+1:

**Comment by core23** [comment]
After thinking a few minutes... An exception would be better, as the other bundles do the same @OskarStark

**Comment by OskarStark** [comment]
:+1:

File: DependencyInjection/SonataUserExtension.php [hunk 1]
```diff
@@ -199,6 +201,8 @@ public function configureClass($config, ContainerBuilder $container)
             $modelType = 'entity';
         } elseif ('mongodb' === $config['manager_type']) {
             $modelType = 'document';
+        } else {
+            throw new \InvalidArgumentException(sprintf('SonataUserBundle - Invalid manager type "%s".', $config['manager_type']));
```
Comment (OskarStark) [inline DependencyInjection/SonataUserExtension.php]: Do we really need the "SonataUserBundle"-prefix?

**Comment by core23** [comment]
Fixed @OskarStark

**Comment by OskarStark** [comment]
:+1: thank you @core23

**Comment by OskarStark** [comment]
@core23 can you merge this in the other branches as well please?

**Pull request merged**